editor/editor_file_system.cpp

```diff
@@ -1719,6 +1729,17 @@ Error EditorFileSystem::_reimport_group(const String &p_group_file, const Vector
 				f->store_line("type=\"" + importer->get_resource_type() + "\"");
 			}
 
+			ResourceUID::ID uid = ResourceUID::INVALID_ID;
+			if (source_file_uids.has(file)) {
+				uid = source_file_uids[file];
+			}
+
+			if (uid == ResourceUID::INVALID_ID) {
+				uid = ResourceUID::get_singleton()->create_id();
+			}
+
+			f->store_line("uid=\"" + ResourceUID::get_singleton()->id_to_text(uid) + "\""); //store in readable format
```

Comment:
Suggested change:
```
f->store_line("uid=\"" + ResourceUID::get_singleton()->id_to_text(uid) + "\""); // Store in readable format.
```

Reply:
I copy/pasted this line from somewhere else in the file that had the same comment, but I can update this one to match the comment style guide yes. I'll do that later today